art/estimators/generation/tensorflow.py

```diff
@@ -185,17 +177,99 @@ def compute_loss(self, x: "np.ndarray", y: "np.ndarray", **kwargs) -> "np.ndarra
     @property
     def model(self) -> "tf.Tensor":
         """
-        Returns the generator tensor.
-
         :return: The generator tensor.
         """
         return self._model
 
     @property
     def encoding_length(self) -> int:
         """
-        Returns the length of the encoding size output.
+        :return: The length of the encoding size output.
+        """
+        return self._encoding_length
+
+
+class TensorFlow2Generator(GeneratorMixin, TensorFlowV2Estimator):  # lgtm [py/missing-call-to-init]
+    """
+    This class implements a GAN with the TensorFlow framework.
+    """
+
+    estimator_params = (
+        TensorFlowV2Estimator.estimator_params
+        # + GeneratorMixin.estimator_params
+        + [
+            "encoding_length",
+        ]
+    )
+
+    def __init__(
+        self,
+        encoding_length: "int",
+        model: "tf.Tensor",
+        channels_first=False,
```

Comment:
Please add typing.

Reply:
I'm confused about this one, is there any typing missing here? or were you referring to another line?

Reply:
Yes, please add typing for `channels_first`. It should be `channels_first: bool = False,`

Reply:
Yes, please add typing for `channels_first`. It should be `channels_first: bool = False,`